aep/general/0142/aep.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 id: 142
 state: approved
+slug: time
```

Comment:
I feel mixed about this one.  "time-and-duration" doesn't seem great either, but "time" feels like it somehow loses something.  And without the "and duration" part, "time" seems potentially too vague/ambiguous?

But I don't have a concrete suggestion for a better slug here, so maybe it's fine.

Reply:
similar to enums, I'll just have time and duration redirect here for now.